playwright-ct.config.ts

```diff
@@ -0,0 +1,46 @@
+import { defineConfig, devices } from "@playwright/experimental-ct-react";
```

Comment:
do we need a separate config, or can we just use the same config? (are there reasons to separate this?)

Reply:
I would have thought so, but the Playwright docs say you should have separate configs for e2e and component tests: https://playwright.dev/docs/test-components#q-can-i-use-playwrighttest-and-playwrightexperimental-ct-reactsveltevuesolid